cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -1024,3 +996,75 @@ func (s *Server) GetInstalledPackageResourceRefs(ctx context.Context, request *c
 		}, nil
 	}
 }
+
+func (s *Server) AddPackageRepository(ctx context.Context, request *corev1.AddPackageRepositoryRequest) (*corev1.AddPackageRepositoryResponse, error) {
+	log.Infof("+helm AddPackageRepository [%v]", request)
+	if request == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "no request provided")
+	}
+	if request.Context == nil || request.Context.Namespace == "" {
```

Comment:
this is just a suggestion, but if the namespace is not provided, we could default it to the global namespace.
this makes it easier for users as the global namespace is not required to be known

Reply:
Yep, it makes sense. Will add it.